lib/sprockets/sass_importer.rb

```diff
@@ -1,2 +1,9 @@
-# Deprecated: Require sprockets/sass_processor instead
-require 'sprockets/sass_processor'
+
+require 'sprockets/autoload'
+
+module Sprockets
+  class SassImporter < Autoload::Sass::Importers::Filesystem
+    # NOTE: Hack to support sourcemap generation in Sass-3.3
+    def public_url(a, b);'';end
```

Comment:
I mean, is not this information useful for sass 3.4.0 users? Because right now we are returning a blank string. But seems it 3.4.0 checks too https://github.com/sass/sass/blob/3.4.0/lib/sass/engine.rb#L359